osquery/core/conversions.cpp

```diff
@@ -316,49 +315,39 @@ bool JSON::valueToBool(const rj::Value& value) {
   return false;
 }
 
-std::string base64Decode(const std::string& encoded) {
-  std::string is;
-  std::stringstream os;
+std::string base64Decode(std::string encoded) {
+  boost::trim_right_if(encoded, boost::is_any_of("="));
+  boost::erase_all(encoded, "\r\n");
+  boost::erase_all(encoded, "\n");
```

Comment:
It's not a good idea trimming '=' from the end of the string before removing '\n' and '\n\r' :)